samples/interstitialSample/src/main/java/com/inmobi/interstitial/sample/InterstitialAdsActivity.java

```diff
@@ -162,6 +161,5 @@ public void onAdImpression(@NonNull InMobiInterstitial inMobiInterstitial) {
                         Log.d(TAG, "onAdImpression");
                     }
                 });
-        mInterstitialAd.show();
```

Comment:
why is this removed?

Reply:
@neelabh-parui-inmobi 

I have moved the show from here to click of the button. Show gets called before load success otherwise